tools/cldr-apps/src/main/java/org/unicode/cldr/web/DBUtils.java

```diff
@@ -618,59 +648,57 @@ private DBUtils() {
         System.err.println("Loading datasource: java:comp/env " + JDBC_SURVEYTOOL);
         ElapsedTimer et = new ElapsedTimer();
 
+        try {
+            final DataSource myOtherDB = InitialContext.doLookup("java:comp/DefaultDataSource");
 
-        if (RESOURCE_HELPER.myDB != null) {
-            datasource = RESOURCE_HELPER.myDB;
-            System.err.println("Got default datasource: " + datasource.toString());
-        } else {
-            System.err.println("myDB was null..");
-            try {
-                final DataSource myOtherDB = InitialContext.doLookup("java:comp/DefaultDataSource");
-
-                if(myOtherDB != null) {
-                    System.err.println("DefaultDataSource was ok");
-                    datasource = myOtherDB;
-                } else {
-                    System.err.println("DefaultDataSource was not OK");
-                    Context initialContext = new InitialContext();
-                    Context eCtx = (Context) initialContext.lookup("java:comp/env");
-                    datasource = (DataSource) eCtx.lookup(JDBC_SURVEYTOOL);
-                }
+            if(myOtherDB != null) {
+                System.err.println("DefaultDataSource was ok");
+                datasource = myOtherDB;
```

Comment:
could we avoid using System.err for non-errors?

Reply:
Actually, we should be using a real logger. I thought about this the other day, and I see there's already an issue filed for it: https://unicode-org.atlassian.net/browse/CLDR-8581

Anyway, I will correct some of these.